src/dropdownToggle/dropdownToggle.js

```diff
@@ -90,6 +90,16 @@ angular.module('mm.foundation.dropdownToggle', [ 'mm.foundation.position', 'mm.f
             }
           };
           $document.on('click', closeMenu);
+
+          angular.element($window).bind('resize', function() {
```

Comment:
We need to clean up this event listener on `$destroy`. See https://code.angularjs.org/1.4.7/docs/guide/directive#creating-a-directive-that-manipulates-the-dom